samples/AzCopyCore/AzCopyCore/AzCopyCore.csproj

```diff
@@ -3,7 +3,9 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+	<!--
```

Comment:
Why is this commented out? This should be the same as https://github.com/dotnet/corefxlab/blob/master/SharedRuntimeVersion.txt (or import the common.props file).

Reply:
It's @terrajobst's fault. :-) I removed it now.